swtbot_test/src/iotLTStestsuites/ObjectGUIAddRemoveObject.java

```diff
@@ -116,7 +116,23 @@ public void tc_07_checkStreamBufferUI() throws Exception {
 	
 	@Test
 	public void tc_08_checkTaskUI() throws Exception {
-		Utility.CheckTaskUI();
```

Comment:
Q: Is this utility deprecated?

Reply:
No, it is used in other RTOS project, but for IoT LTS case, this utility is not correct anymore, because there is a default task in newest IoT LTS package now. Therefore, we are no longer using this utility here

Reply:
Why not create a different utility for IoT LTS instead of hardcode, next time if the package is updated you will need to modify every test case that has the new code.

Reply:
I have fixed this in my latest patchset